src/models/modPingSettings.ts

```diff
@@ -0,0 +1,37 @@
+import { DataTypes, Model } from 'sequelize';
+import { sequelize } from '@/sequelize-instance';
+import type { InferAttributes, InferCreationAttributes } from 'sequelize';
+
+export class ModPingSettings extends Model<InferAttributes<ModPingSettings>, InferCreationAttributes<ModPingSettings>> {
+    declare user_id: string;
+    declare online: boolean;
```

Comment:
These are still using spaces.

Reply:
thanks vscode (i changed the indent setting after making this)